pkg/iscsi/v1alpha1/mount.go

```diff
@@ -46,7 +47,13 @@ func ResizeVolume(volumePath string) error {
 			if err := util.ReScan(); err != nil {
 				return err
 			}
-			if err := util.ReSizeFS(mpt.Device); err != nil {
+			switch fsType {
+			case "ext4":
+				err = util.ResizeExt4(mpt.Device)
+			case "xfs":
+				err = util.ResizeXFS(volumePath)
```

Comment:
What about other types? Should we return an error `not supported or manual op needed`?